feature-libs/cart/base/core/store/actions/cart.action.ts

```diff
@@ -58,10 +58,15 @@ interface CreateCartFailPayload extends CreateCartPayload {
   error: any;
 }
 
-export class CreateCartFail extends StateUtils.EntityFailAction {
+export class CreateCartFail
+  extends StateUtils.EntityFailAction
+  implements ErrorAction
+{
   readonly type = CREATE_CART_FAIL;
+  error: any;
   constructor(public payload: CreateCartFailPayload) {
     super(MULTI_CART_DATA, payload.tempCartId, payload.error);
+    this.error = payload.error;
```

Comment:
I can see many occurences of :
- declaring property `error: any` explicitly in the child class
- `this.error = <something from constructor argument>` in the child class
... while it doens't seem to be needed, thanks to the super class which does the job

Reply:
Apologies for overlooking these places. Done!